lib/osf-components/addon/components/editable-field/license-manager/component.ts

```diff
@@ -31,22 +33,22 @@ export default class LicenseManagerComponent extends Component implements Licens
     // private
     @service analytics!: Analytics;
     @service intl!: Intl;
-    @service store!: DS.Store;
+    @service store!: Store;
     @service toast!: Toast;
 
-    changeset!: ChangesetDef;
-    requestedEditMode: boolean = false;
+    changeset!: BufferedChangeset;
+    requestedEditMode = false;
 
-    showText: boolean = false;
-    licensesAcceptable!: QueryHasManyResult<License>;
-    helpLink: string = 'https://help.osf.io/hc/en-us/articles/360019739014-Licensing';
-    currentLicense!: License;
+    showText = false;
+    licensesAcceptable!: QueryHasManyResult<LicenseModel>;
+    helpLink = 'https://help.osf.io/hc/en-us/articles/360019739014-Licensing';
+    currentLicense!: LicenseModel;
     currentNodeLicense!: NodeLicense;
-    selectedLicense!: License;
+    selectedLicense!: NodeModel['license'];
```

Comment:
Correct. It's equivalent to `DS.PromiseObject<LicenseModel> & LicenseModel` which is the type of license `field` of `NodeModel`. This change was necessary because we are setting the nodeLicense on the node [here](https://github.com/CenterForOpenScience/ember-osf-web/pull/1203/files/353132765b65c20f3029ba1668f124436036d08f#diff-72da45bfddee9c8536445e16b64fa21867a91f00130523bb2c938f2886899f74R135) `NodeModel['license']` because it avoids `import DS from 'ember-data'` which is deprecated. See https://github.com/emberjs/rfcs/blob/master/text/0395-ember-data-packages.md for more context.